helm_deploy/dps-data-compliance/templates/_envs.tpl

```diff
@@ -55,6 +58,11 @@ env:
     value: "{{ .Values.env.IMAGE_RECOGNITION_UPLOAD_PERMITS_PER_SECOND }}"
   {{- end }}
 
+  {{- if .Values.env.PRISON_API_OFFENDER_IDS_PAGE_LIMIT }}
```

Comment:
Unrelated, just noticed this was missing since it's been using the default value so far in all envs.